Sources/StreamVideo/WebRTC/WebRTCClient.swift

```diff
@@ -154,8 +163,9 @@ class WebRTCClient: NSObject, @unchecked Sendable {
     private(set) var signalChannel: WebSocketClient?
     
     private(set) var sessionID: String
-    private var token: String
     
+    private var token: String = ""
```

Comment:
This is not ideal, but I guess it's the consequence of the self isolation.

Reply:
not sure I understand the last part - how can you attempt to join a call without calling join? If that's somehow possible, we should try not to crash.